emails/views.py

```diff
@@ -556,9 +570,19 @@ def _handle_bounce(message_json):
 
 
 def _get_text_html_attachments(message_json):
-    bytes_email_message = message_from_bytes(
-        message_json['content'].encode('utf-8'), policy=policy.default
-    )
+    if 'content' in message_json:
+        message_content = message_json['content'].encode('utf-8')
+    elif 'receipt' in message_json and 'action' in message_json['receipt']:
+        if 'S3' in message_json['receipt']['action']['type']:
+            bucket = message_json['receipt']['action']['bucketName']
+            object_key = message_json['receipt']['action']['objectKey']
```

Comment:
These 3 lines are duplicated up in `_sns_message`. Could we put them in a helper function?
```
bucket, object_key = _get_bucket_and_key_from_s3_json(message_json)
```
?